modules/olm-checking-deleted-operators.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-checking-deleted-operators-{context}']
+= Checking deleted Operators
+Although a deleted Operator subscription is no longer receiving updates, all traces of that Operator are not automatically removed. Here is how you can see the current state of the deleted Operator:
+
+* Go to Catalog → Operator Management → Operator Catalogs. You will see the uninstalled Operator still listed under its namespace, ready for you to select Create Subscription to tell the OLM to subscribe to install it again.
+
+* Go to OperatorHub in the namespace. It will show the Operator as not installed, since it is looking for the Operator’s subscription and not its listing under Operator Management. So the Operator is still partially installed.
+
+If you try to install the Operator again, it would not add a new entry to
+the Operator Management screen since the Operator is still there. But the
+bits under the Operator are not being updated. If you want to remove all
```

Comment:
@aravindhp Is this true?
I suspect that if grpc pods serving marketplace are refreshed every hour, that this includes grpc pods for installed operators.

Reply:
If the `CatalogSourceConfig` still exists then the `CatalogSource` will get the updates from Quay on the next hourly sync.